qutebrowser/browser/urlmarks.py

```diff
@@ -141,12 +141,33 @@ class QuickmarkManager(UrlMarkManager):
         - removed gets emitted with the name as argument.
     """
 
+    def _quickmark_directory(self):
+        """Return the directory containing the quickmarks.
+
+        For quickmarks are data, return by default the standard data
+        directory. In order not to break the compatibility with an older
+        behavior, if quickmarks are found in the standard configuration
+        directory, return this directory instead.
+        """
+        config_dir = standarddir.config()
+        data_dir = standarddir.data()
+        if config_dir is not None:
+            isinconfigdir = os.path.isfile(
+                os.path.join(config_dir, 'quickmarks')
```

Comment:
Here you need to assemble the full file path to make the decision, return the directory name and later assemble the file path again, which is somewhat useless. Can't this method return the full path to the `quickmarks` file directly?